src/components/TodoList/TodoList.tsx

```diff
@@ -16,85 +28,80 @@ export const TodoList: React.FC = () => (
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
-
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">8</td>
-        <td className="is-vcentered">
-          <span className="icon" data-cy="iconCompleted">
-            <i className="fas fa-check" />
-          </span>
-        </td>
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-success">quo adipisci enim quam ut ab</p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
+          </td>
+          <td className="is-vcentered is-expanded">
+            <p
+              className={classNames({
+                'has-text-success': todo.completed,
+                'has-text-danger': !todo.completed,
+              })}
+            >
+              {todo.title}
+            </p>
+          </td>
+          <td className="has-text-right is-vcentered">
+            <button
+              data-cy="selectButton"
+              className="button"
+              type="button"
+              onClick={() => {
+                setSelectedTodo(todo);
+              }}
+            >
+              <span className="icon">
+                <i
+                  className={classNames({
+                    'far fa-eye-slash': todo.id === todoId,
+                    'far fa-eye': todo.id !== todoId,
+                  })}
+                />
+              </span>
+            </button>
+          </td>
+        </tr>
+      ) : (
+        <tr data-cy="todo" className="" key={todo.id}>
+          <td className="is-vcentered">{todo.id}</td>
+          <td className="is-vcentered" />
+          <td className="is-vcentered is-expanded">
+            <p
+              className={classNames({
+                'has-text-success': todo.completed,
+                'has-text-danger': !todo.completed,
+              })}
+            >
+              {todo.title}
+            </p>
+          </td>
+          <td className="has-text-right is-vcentered">
+            <button
+              data-cy="selectButton"
+              className="button"
+              type="button"
+              onClick={() => {
+                setSelectedTodo(todo);
+              }}
+            >
+              <span className="icon">
+                <i
+                  className={classNames({
+                    'far fa-eye-slash': todo.id === todoId,
+                    'far fa-eye': todo.id !== todoId,
+                  })}
+                />
+              </span>
+            </button>
+          </td>
+        </tr>
+      )))}
+      ;
```

Comment:
delete this, it doesn't has to be on the page
![image](https://github.com/mate-academy/react_dynamic-list-of-todos/assets/133110903/f592f6b0-4ee8-4f45-93f2-001f973315c8)